dist/s.js

```diff
@@ -8,7 +8,14 @@
   const envGlobal = hasSelf ? self : global;
 
   let baseUrl;
-  if (typeof location !== 'undefined') {
+
+  if (typeof document !== 'undefined') {
+    const baseEl = document.head.querySelector('base[href]');
```

Comment:
Same feedback as in https://github.com/guybedford/es-module-shims/pull/26.

Reply:
Done!